frontend/src/features/mapSideBar/hooks/useQueryParamSideBar.tsx

```diff
@@ -67,6 +69,9 @@ const useQueryParamSideBar = (formikRef?: any): IMapSideBar => {
         ? +searchParams.buildingId
         : undefined,
     );
+    setAssociatedParcelId(
+      searchParams.associatedParcelId ? +searchParams.associatedParcelId || undefined : undefined,
```

Comment:
I think this can be replaced with `searchParams.associatedParcelId ? +searchParams.associatedParcelId 
 : undefined`